core/themes/basis/css/component/node.css

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file
+ * Styling for node content.
+ */
+
+.node {
+  position: relative;
+}
+
+/* Display "Draft" watermark on unpublished content. */
+/* See basis_preprocess_node(). */
+.node[data-status]:before {
+  content: attr(data-status);
+  position: absolute;
+  z-index: 5;
+  width: 100%;
+  top: 20%;
+  pointer-events: none;
+  text-align: center;
+  text-transform: uppercase;
+  font-size: 10em;
```

Comment:
The other potential issue is if the word ends up being something other than draft, it could be wider than these styles account for. I can't think of a good solution for that issue :/ I think erring on the side of a smaller font may be safer.